wiki/geyser/faq.mdx

```diff
@@ -150,7 +150,7 @@ Here you can find answers to frequently asked questions about Geyser - if you ha
     <Q>
         Sometimes, when I go really far out, the world starts to experience very odd visual bugs...
         <A>
-            This is a Bedrock client issue caused by Bedrock being 32-bit and Java being 64-bit. For more information read [here](https://minecraft.wiki/w/Bedrock_Edition_distance_effects).
+            This is a Bedrock client issue caused by Bedrock being 32-bit and Java being 64-bit. For more information read [here](https://minecraft.wiki/w/Bedrock_Edition_distance_effects). This can be workaround using an [unofficial extension](https://github.com/oryxel1/GeyserFloatingPoints) by spoofing your position.
```

Comment:
```suggestion
            This is a Bedrock client issue caused by Bedrock being 32-bit and Java being 64-bit. For more information read [here](https://minecraft.wiki/w/Bedrock_Edition_distance_effects). This can be worked around using an [unofficial Geyser extension](https://github.com/oryxel1/GeyserFloatingPoints) that spoofs the player position.
```